t/cobrand/highwaysengland.t

```diff
@@ -1,11 +1,22 @@
 use FixMyStreet::TestMech;
 use FixMyStreet::App;
+use FixMyStreet::Script::Reports;
+use FixMyStreet::Cobrand::HighwaysEngland;
+use HighwaysEngland;
+use Test::MockModule;
+
+my $he_mock = Test::MockModule->new('HighwaysEngland');
+$he_mock->mock('database_file', sub { FixMyStreet->path_to('t/geocode/roads.sqlite'); });
 
 # disable info logs for this test run
 FixMyStreet::App->log->disable('info');
 END { FixMyStreet::App->log->enable('info'); }
 
 
+my $he_mock_cobrand = Test::MockModule->new('FixMyStreet::Cobrand::HighwaysEngland');
+$he_mock_cobrand->mock('report_new_is_on_he_road', sub { 1 } );
```

Comment:
I don't understand what this line is for; this function is only called on .com/council, but you're only testing the HE cobrand here, unless I've misunderstood.